tests/data/ciscosb_cbs350-4x.json

```diff
@@ -12058,7 +12058,7 @@
                     "sensor_oid": ".1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.10",
                     "sensor_index": "1.10",
                     "sensor_type": "ciscosb",
-                    "sensor_descr": "GigabitEthernet1/0/10 PoE",
+                    "sensor_descr": "PoE",
```

Comment:
Hi @rudybroersma 
Seems that the descr is now "PoE" for all sensors. Could you fix that ?

Reply:
Problem is, I haven't changed that.  I'm only reading "CISCOSB-rlInterfaces::swIfOperSuspendedStatus" which has nothing to do with PoE.

If I add a debug line to my code, like this:

`            //Discover Sensors
            discover_sensor(null, 'state', $device, $cur_oid . $index, $index, $state_name, $descr, 1, 1, null, null, null, null, $entry, 'snmp', $index);
echo "DESCRIPTION: $descr\n";
`

I get:
State: DESCRIPTION: GigabitEthernet1/0/1 Suspended Status
DESCRIPTION: GigabitEthernet1/0/2 Suspended Status
DESCRIPTION: GigabitEthernet1/0/3 Suspended Status
DESCRIPTION: GigabitEthernet1/0/4 Suspended Status
DESCRIPTION: GigabitEthernet1/0/5 Suspended Status
DESCRIPTION: GigabitEthernet1/0/6 Suspended Status

My code seems fine.

Worse, the lack of interface description seems to be coming from the LibreNMS test suite?
My own LibreNMS installation generates the descriptions just fine:

`
MariaDB [librenms]> select sensor_class, sensor_oid, sensor_index, sensor_type, sensor_descr, sensor_current from sensors where device_id = 190 ;
+--------------+---------------------------------------+--------------+-------------------------+------------------------------------------+----------------+
| sensor_class | sensor_oid                            | sensor_index | sensor_type             | sensor_descr                             | sensor_current |
+--------------+---------------------------------------+--------------+-------------------------+------------------------------------------+----------------+
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.7  | 1.7          | ciscosb                 | GigabitEthernet1/0/7 PoE                 |          6.852 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.8  | 1.8          | ciscosb                 | GigabitEthernet1/0/8 PoE                 |          7.323 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.9  | 1.9          | ciscosb                 | GigabitEthernet1/0/9 PoE                 |          7.148 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.10 | 1.10         | ciscosb                 | GigabitEthernet1/0/10 PoE                |          7.194 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.11 | 1.11         | ciscosb                 | GigabitEthernet1/0/11 PoE                |          7.133 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.13 | 1.13         | ciscosb                 | GigabitEthernet1/0/13 PoE                |          9.338 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.14 | 1.14         | ciscosb                 | GigabitEthernet1/0/14 PoE                |          8.719 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.15 | 1.15         | ciscosb                 | GigabitEthernet1/0/15 PoE                |          8.603 |
| power        | .1.3.6.1.4.1.9.6.1.101.108.1.1.5.1.16 | 1.16         | ciscosb                 | GigabitEthernet1/0/16 PoE                |          8.985 |
`

Reply:
yes, for some reason, your test-data generation is giving funny results.

Reply:
> yes, for some reason, your test-data generation is giving funny results.

Yeah, any ideas what might cause this and how to fix this? It's frustrating :-)

Reply:
Do you use the `DBTEST=1` statement in your `.env` file when you generate the JSON ? 
I personnally do : 
- Add or uncomment the `DBTEST=1` statement in the .env file.
- Run `lnms dev:check -o <anyOsYouWant>` up until necessary dependencies are installed, and DB is reset, and I stop the command:
```
  import pkg_resources
Only checking os:  <anyOsYouWant>
Running unit check... 
Refreshing database...done
^C
```
- Run the `save-test-data.php` script. 

This seems to give consistant results, so far.